internal/tests/cli/boundary/target.bats

```diff
@@ -4,6 +4,7 @@ load _auth
 load _connect
 load _targets
 load _helpers
+load _target_host_sources
```

Comment:
Not sure if there is a convention for the order of these

Reply:
Just scanning through other `bats` files I have not noticed a convention nor any implications from switching the order for `targets.bats`